refs/biblio.json

```diff
@@ -186,9 +186,9 @@
             "Aharon Lanin",
             "Andrew Glass"
         ],
-        "href": "http://www.unicode.org/reports/tr9/",
+        "href": "http://www.unicode.org/reports/tr9/tr9-35.html",
         "publisher": "Unicode Consortium",
-        "rawDate": "2014-06-05",
+        "date": "18 May 2016",
```

Comment:
fyi, `rawDate` gets converted to `date` by the app. It's just more machine readable.